code/modules/client/loadout/loadout_general.dm

```diff
@@ -10,6 +10,12 @@
 	path = /obj/item/dice/d20
 	cost = 300
 
+/datum/gear/misc/eightball
+	display_name = "Magic Eightball"
+	description = "A black ball with a stenciled number eight in white on the side. It seems full of dark liquid. \n The instructions state that you should ask your question aloud, and then shake."
```

Comment:
Remove the `\n` and extra space.

Reply:
Oh dangit. The original commit had but but I accidentally did it in main instead of a branch.